en/test_and_ci/unit_tests.md

```diff
@@ -0,0 +1,128 @@
+# Unit Tests
+
+PX4 provides a simple base [Unittest-class](https://github.com/PX4/Firmware/blob/master/src/include/unit_test.h). Each developer is encouraged to write unit tests in the process of adding a new feature to the PX4 framework.
+
+## Writing a Test
+
+1. Create a new .cpp file within [tests](https://github.com/PX4/Firmware/tree/master/src/systemcmds/tests) with name `test_[description].cpp`. 
+
+2. Within `test_[description].cpp` include the base unittest-class `<unit_test.h>` and all files required to write a test for the new feature. 
```

Comment:
Just FYI, you can number all of these as "1." and gitbook will automatically handle correct numbering (I think github will too). It isn't a big deal, but if you need to reorder the list at some point it makes things easier.